sunpy/util/util.py

```diff
@@ -92,12 +92,14 @@ def expand_list(inp):
     ----------
     * https://stackoverflow.com/questions/2185822/expanding-elements-in-a-list/2185971#2185971
     """
-    return [item for item in expand_list_generator(inp)]
+    return list(expand_list_generator(inp))
 
 
 def expand_list_generator(inp):
+    exclude_union = (str | bytes)
+    include_union = (list | tuple | Iterator)
```

Comment:
Version 3

Reply:
It's good if it's working, right? or I'm missing something

Reply:
Understanding why it works is what I need to learn.